webapp/sdf/src/components/item-component.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <h2>{{ item.title }} ({{ item.votes }})</h2>
+    <button v-on:click="updateItem(item, 1)">Upvote</button>
+    <button @click="updateItem(item, -1)">Downvote</button>
+    <button v-on:click="removeItem(item)">Remove</button>
+  </div>
+</template>
+
+<script>
+import Item from "../classes/item";
+export default {
+  name: "ItemComponent",
+  props: {
+    item: Item,
+  },
+  methods: {
+    removeItem: function (_item) {
+      this.$emit("removeEvent", _item);
+    },
+
+    updateItem: function (_item, value) {
```

Comment:
```suggestion
    updateItem: function (item, value) {
```
Using _underscore prefix for an argument is a naming convention when you don't use the argument in the function body, but you *do use* it on line 23. Other naming conventions use underscores to indicate private methods/properties. But that doesn't seem to be the case here. See: https://eslint.org/docs/rules/no-underscore-dangle